test/boost/mvcc_test.cc

```diff
@@ -451,6 +452,7 @@ SEASTAR_TEST_CASE(test_apply_to_incomplete) {
             table.add_row(m, ck1, "v");
             e += m;
             assert_that(table.schema(), e.squashed()).is_equal_to(mutation_partition::make_incomplete(s));
+            e.evict();
```

Comment:
Why do you need this? The destructor of mvcc_partition does this too (as it should).

Reply:
Right. After I added evict() to the destructor, these are not needed anymore. Fixed.